pkg/application/options.go

```diff
@@ -262,6 +262,20 @@ func WithLoggerTagsFromConfig(app *App) {
 	})
 }
 
+func WithLoggerVersionTag(version string) Option {
```

Comment:
where is this requirement coming from?

Reply:
from me :-) private project where the user of the app should somehow be able to know whether he is using an up-to-date version without the app having to call an api endpoint to determine that.